src/main/java/net/sf/jabref/gui/PreviewPanel.java

```diff
@@ -381,22 +375,6 @@ public void actionPerformed(ActionEvent e) {
         }
     }
 
-    class CopyPreviewAction extends AbstractAction {
-
-        public CopyPreviewAction() {
-            super(Localization.lang("Copy preview"), IconTheme.JabRefIcon.COPY.getSmallIcon());
-            putValue(Action.SHORT_DESCRIPTION, Localization.lang("Copy preview"));
-            putValue(Action.ACCELERATOR_KEY, Globals.getKeyPrefs().getKey(KeyBinding.COPY_PREVIEW));
-        }
-
-        @Override
-        public void actionPerformed(ActionEvent e) {
-            previewPane.selectAll();
-            previewPane.copy();
```

Comment:
Do you think, it's possible to adapt `COPY_CITATION_HTML` to copy the text of JabRef's "Preview" style?